ocaml/idl/datamodel_errors.ml

```diff
@@ -1940,7 +1940,15 @@ let _ =
   error Api_errors.invalid_repository_proxy_url ["url"]
     ~doc:"The repository proxy URL is invalid." () ;
   error Api_errors.invalid_repository_proxy_credential []
-    ~doc:"The repository proxy username/password is invalid." ()
+    ~doc:"The repository proxy username/password is invalid." () ;
+  error Api_errors.apply_livepatch_failed ["msg"]
+    ~doc:"Failed to apply a livepatch." () ;
+  error Api_errors.livepatch_build_id_mismatched []
+    ~doc:"The build id of livepatch does not match with running component." () ;
```

Comment:
Grammar could be improved: _The build ID of the livepatch does not match the running component_